ftw/keywordwidget/vocabularies.py

```diff
@@ -0,0 +1,44 @@
+from binascii import b2a_qp
+from plone.dexterity.utils import safe_unicode
```

Comment:
Unfortunately `safe_unicode` and `safe_utf8` are only available as of `plone.dexterity` `2.4.0`, we're still on `2.1.3` 😞  and probably can't upgrade to `2.4.0` in the near future. Could you adapt these imports to use `Products.CMFPlone.utils` instead?